pkg/cvo/cvo.go

```diff
@@ -691,14 +691,14 @@ func (b *resourceBuilder) builderFor(m *lib.Manifest, state payload.State) (reso
 	if resourcebuilder.Mapper.Exists(m.GVK) {
 		return resourcebuilder.New(resourcebuilder.Mapper, config, *m)
 	}
-	client, err := dynamicclient.New(config, m.GVK, m.Object().GetNamespace())
+	client, err := dynamicclient.New(config, m.GVK, m.Obj.GetNamespace())
```

Comment:
I don't understand this change.  Did we loose the getter somewhere?  Although not clear to me why we'd have a getter in front of a public property...

Reply:
Comment from when I merged `manifest` into library-go. The method isn't needed: https://github.com/openshift/library-go/pull/680#discussion_r385556651